Dockerfile

```
@@ -33,11 +33,16 @@ LABEL maintainer="Operate First" \
     version.ksops="${KSOPS_VERSION}" \
     version.sops="${SOPS_VERSION}"
 
+ADD ./labels.yaml /etc/config/labels.yaml
+ADD ./labels.yaml /gen-source/labels.yaml
+
+RUN ls /gen-source/
```

```suggestion
RUN \
   curl -o /etc/config/labels.yaml https://raw.githubusercontent.com/operate-first/common/main/labels.yaml && \
   cp /etc/config/labels.yaml  /gen-source/labels.yaml
```

we can do this way. move this run command is already present run if this not needed before COPY.